lib/rollbax/item.ex

```diff
@@ -27,7 +27,7 @@ defmodule Rollbax.Item do
     if map_size(meta) == 0 do
       data
     else
-      Map.put(data, "custom", meta)
+      Map.merge(data, meta)
```

Comment:
What a twist, I've just checked how data will be displayed for `Map.merge/2` case.
Seems it wont be discarded and displays nicely. It's not what documentation says though, and might be fragile count on it.
I'm torn between 2 options:

* let any `meta` be merged with `data` as is.
It makes possible to insert `person`, etc. sections into payload, but at the same time end user can easily get into trouble:
```elixir
Rollbax.report(.., .., %{uuid: "xyz"})
```
this will affect `uuid` of Item and might be not what user wants
* pop predefined keys from metadata like in the sample code above, still has `uuid` ambiguity issue.